packages/autorest.typescript/test/integration/generated/appconfiguration/src/models/index.ts

```diff
@@ -68,6 +68,20 @@ export interface Label {
   readonly name?: string;
 }
 
+/** The result of a list request. */
+export interface KeyListResult extends TestParentList {
+  /** The roll out count summary of the savings plans */
+  summary?: string;
+  /** The collection value. */
+  items?: Key[];
+  /** The URI that can be used to request the next set of paged results. */
+  nextLink?: string;
+  /** The collection value. */
+  items?: Key[];
+  /** The URI that can be used to request the next set of paged results. */
+  nextLink?: string;
+}
```

Comment:
After debugging I found this is not js codegen issue but a m4 issue which gives us duplicated nextLink and items properties.
![image](https://github.com/user-attachments/assets/a55f127d-6b75-4226-ad95-4bb75c928de2)

Reply:
@kazrael2119 Could you open an issue in [autorest side](https://github.com/Azure/autorest/issues) and remove swagger case for now?

Reply:
https://github.com/Azure/autorest/issues/5012